src/cart/services/cart.service.ts

```diff
@@ -1,25 +1,36 @@
 import { Injectable } from '@nestjs/common';
+import { randomUUID } from 'crypto';
+import { Cart, CartStatuses } from '../models';
+import { PutCartPayload } from 'src/order/type';
 
-import { v4 } from 'uuid';
+function getDate(date = new Date()) {
+  date.setHours(0);
+  date.setMinutes(0);
+  date.setMinutes(0);
+  date.setMilliseconds(0);
 
-import { Cart } from '../models';
+  return (date.getTime() / 1000).toString();
+}
 
 @Injectable()
 export class CartService {
   private userCarts: Record<string, Cart> = {};
 
   findByUserId(userId: string): Cart {
-    return this.userCarts[ userId ];
+    return this.userCarts[userId];
   }
 
-  createByUserId(userId: string) {
-    const id = v4();
+  createByUserId(user_id: string): Cart {
     const userCart = {
-      id,
+      id: randomUUID(),
+      user_id,
+      created_at: getDate(),
```

Comment:
i wonder, why do you need to store date as string and just actual timestamp (Date.now()) ?

Reply:
You are right, I forgot update that